contrib/Cargo.latest.lock

```diff
@@ -18,7 +18,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "73290177011694f38ec25e165d0387ab7ea749a4b81cd4c80dae5988229f7a57"
 
 [[package]]
-name = "bitcoin_hashes"
+name = "hashes"
```

Comment:
In 4d10fd86afe0cc63b38881a18648c9a1b69b7250:

How did you make these changes to the lockfiles? They don't seem to be valid.

Reply:
Intereting, how the CI work?

Reply:
We don't actually test these lockfiles in CI. It looks like these were introduced in https://github.com/rust-bitcoin/rust-secp256k1/pull/591 as an early version of what we did in rust-bitcoin, and it didn't occur to us(?) to actually test them.

I guess we should update CI, and maybe we should rename the files to match the Cargo-recent.lock and Cargo-minimal.lock names from rust-bitcoin.

Reply:
Want me to pick that up @sanket1729?

Reply:
oh I didn't notice those lock files, I'll rename as suggested and put them in the crate root like in rust-bitcoin.